.storybook/components/TokenSpecimen/TokenSpecimen.js

```diff
@@ -21,6 +21,7 @@ export class TokenSpecimen extends Component {
           }
           contentEditable
           style={this.props.inlineStyles}
+          suppressContentEditableWarning={true}
```

Comment:
since `contentEditable` is implied truthy, nit:
```suggestion
          suppressContentEditableWarning
```

Reply:
Sounds good, will do, thanks!